opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/SnmpConfigRestService.java

```diff
@@ -121,12 +122,12 @@ protected void tearDown() {
     @GET
     @Produces({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON, MediaType.APPLICATION_ATOM_XML})
     @Path("{ipAddr}")
-    public SnmpInfo getSnmpInfo(@PathParam("ipAddr") String ipAddr) {
+    public SnmpInfo getSnmpInfo(@PathParam("ipAddr") String ipAddr, @QueryParam("location")String location) {
```

Comment:
We should document this extra param here: https://github.com/OpenNMS/opennms/blob/develop/opennms-doc/guide-development/src/asciidoc/text/rest/snmp_configuration.adoc